code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -120,6 +119,14 @@
 	last_healed = world.time + heal_interval
 
 /obj/effect/alien/resin/special/pylon/core/attack_alien(mob/living/carbon/Xenomorph/M)
+	if(!warn && world.time < HIVECORE_COOLDOWN_CUTOFF)
```

Comment:
if you are in the process of destroying the core you should not be able to destroy it again (aka no stacking of the do_After)